generators/base/src/utils/parseGeneratorConfig.ts

```diff
@@ -1,20 +1,9 @@
-import { FernGeneratorExec, GeneratorExecParsing } from "@fern-api/browser-compatible-base-generator";
+import { FernGeneratorExec } from "@fern-api/browser-compatible-base-generator";
 import { readFile } from "fs/promises";
 
 export async function parseGeneratorConfig(pathToConfig: string): Promise<FernGeneratorExec.GeneratorConfig> {
     const configStr = await readFile(pathToConfig);
     // biome-ignore lint/suspicious/noConsole: allow console
     console.log(`Parsed ${pathToConfig}`);
-    const rawConfig = JSON.parse(configStr.toString());
-    const parsedConfig = await GeneratorExecParsing.GeneratorConfig.parse(rawConfig, {
-        unrecognizedObjectKeys: "passthrough"
-    });
-
-    if (!parsedConfig.ok) {
-        // biome-ignore lint/suspicious/noConsole: allow console
-        console.log(`Failed to parse ${pathToConfig}`);
-        throw new Error("Failed to parse the generator configuration");
-    }
-
-    return parsedConfig.value;
+    return JSON.parse(configStr.toString()) as FernGeneratorExec.GeneratorConfig;
```

Comment:
you shouldn't have had to make these changes. You should be able to access the `parse` function from the serialization package in the SDK